pitop/processing/speech/text_to_speech.py

```diff
@@ -0,0 +1,76 @@
+import festival
+import os
+
+
+class TTS:
+    __VOICE_DIR = os.path.join(os.sep, "usr", "share", "festival", "voices")
+
+    def __init__(self):
+        self._speed = 1.0
+        self.__voices = self.get_voices()
+        self._language = None
+        self.language = "us"
+        self._voice = None
+        self.voice = self.__voices.get(self.language)[0]
+
+    def __call__(self, text: str):
```

Comment:
I think we should make this available as a `say` method also, particularly for use of the TTS class as a standalone without creating a Pitop object.